include/hip/nvcc_detail/hip_runtime_api.h

```diff
@@ -50,15 +50,64 @@ hipMemcpyHostToHost
 ,hipMemcpyDeviceToDevice
 ,hipMemcpyDefault
 } hipMemcpyKind ;
+#if 0
```

Comment:
Can we remove this instead of using ```#if 0```? We can always revert this commit if we want to go back.